packages/devtools_app/lib/src/charts/treemap.dart

```diff
@@ -745,7 +749,8 @@ class MultiCellPainter extends CustomPainter {
     );
 
     final rectPaint = Paint();
-    rectPaint.color = node.displayColor;
+    final paintColor = node.isDeferred ? Colors.grey : node.displayColor;
```

Comment:
Let's actually move this logic into the `displayColor` getter itself

Reply:
Also instead of `Colors.grey`, let's create a `treemapDeferredColor` variable. Let's try using the `Colors.indigo[100]` from the Material UI spec here: https://api.flutter.dev/flutter/material/Colors-class.html

We can list it along with the other treemap colors here: https://github.com/flutter/devtools/blob/master/packages/devtools_app/lib/src/ui/colors.dart#L74

Reply:
Unresolving to address the color change